sdk/core/azure-core/src/http/curl/curl.cpp

```diff
@@ -446,7 +446,16 @@ void CurlSession::ParseChunkSize(Context const& context)
       if (i > 1 && this->m_readBuffer[index] == '\n')
       {
         // get chunk size. Chunk size comes in Hex value
-        this->m_chunkSize = static_cast<int64_t>(std::stoull(strChunkSize, nullptr, 16));
+        try
+        {
+          this->m_chunkSize = static_cast<int64_t>(std::stoull(strChunkSize, nullptr, 16));
+        }
+        catch (std::invalid_argument& err)
+        {
+          (void)err;
+          // Server can return something like `\n\r\n` as a
```

Comment:
The sentence is not finished, might be merge error.

Reply:
yes, the rebase, I think I missed that one. Thanks